index.html

```diff
@@ -206,8 +206,54 @@ <h2>Skills</h2>
           </div>
         </section>
          </div> 
-     <!-- </section> -->
-
+         <section class="contact-section">
+            <header class="contact-header">
+                <h2>
+                    I'm always interested in hearing about new projects,
+                     so if you'd like to chat please get in touch.
+                </h2>
+            </header>
+            <div class="contact-form">
+                <form action="https://formspree.io/f/mvolbpag" method="post">
+                    <input type="text" name="fullname" id="fullname" maxlength="30" placeholder="Full name">
+                    <input type="email" name="email" id="email" placeholder="Email address">
+                    <textarea name="message" id="textarea" maxlength="500" placeholder="Enter text here"></textarea>
+                    <button class="contact-btn btn-anime" type="submit">Get in touch</button>
```

Comment:
- [ ] Please add an interaction like hover style for the button. Take a look at this design guide for your chosen template to implement the button interaction.
Since you are using the green colour, please stick with it.

Design Guide|
:-----------------:|
![a1](https://user-images.githubusercontent.com/39523312/160482083-82eafbc5-fcf6-4380-834c-8a15e172d789.png)|

Reply:
@mmsesay  i had already implimented hover 
.btn-anime:hover {
  background-color: #36b37f;
}
check line  789 of the CSS file 
the enable is the same color as the  hover (based on the design) which means there's no color change when you hover, kindly explain in clearer terms what change you want me to impliment

Reply:
Hi @kamzzy, I appreciate the fact that you tried to point out the section where your implementation is. However, I have seen you used the right colour. I will suggest you add the cursor pointer to the button when hovered so that it can show some difference a bit.